js/two_block.js

```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
+
+    if(value === 'Mark') {
+        console.log(`Hi, ${value}`)
+    } else {
+        console.log(`Hello, ${value}`)
+    }
+}
+
+helloOleg('Pol');
+
+
+//  2)
+const hypotenuse = function(n, m) {
+
+    const result = (n*n) + (m*m);
+    console.log(Math.sqrt(result));
+}
+
+hypotenuse(4, 4);
+
+
+// 3)
+const min = function(a ,b) {
+    let minValue = 0;
+    let maxValue = 0;
+    if(a < b) {
+        minValue = a;
+        maxValue = b;
+    } else {
+        minValue = b;
+        maxValue = a;
+    }
+
+    console.log(minValue);
+}
+
+min(3, -1)
+
+// 4 )
+const isEven = function(value) {
+    if(value % 2 === 0) {
+        value = 'true';
+    } else {
+        value = 'false';
+    }
+
+    console.log(value);
+}
+
+isEven(98);
+
+// 5)
+const deleteChars = function() {
```

Comment:
your function must apply arg not hardcoded hello

Reply:
Corrected